src/classes/CW_AudienceGroupMemberTriggerHandler.cls

```diff
@@ -0,0 +1,63 @@
+/**
+* File Name :   CW_AudienceGroupMemberTriggerHandler.cls
+* Description : B-106803
+*
+* Modification Log
+* ============================================================================
+* Ver Date	   Author		 Modification
+* --- ---------- -------------- --------------------------
+* 0.1 15/04/2014 Klea Kolaric   Initial version
+*
+*
+*/
+public with sharing class CW_AudienceGroupMemberTriggerHandler {
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.